dist/s_style

```diff
@@ -32,6 +32,7 @@ if [ "$single_file" -ne 1 ]; then
     sed -e '/Makefile.in/d' \
         -e '/test\/3rdparty/d' \
         -e '/test\/checksum\/power8/d' \
+        -e '/bench\/workgen\/workgen_wrap.cxx/d' \
```

Comment:
Why are we excluding `workgen_wrap.cxx` here? If it's due to the `.cxx` rather than `.cpp` extension I think we just missed that during [WT-10458](https://jira.mongodb.org/browse/WT-10458)

Reply:
It's because workgen_wrap is auto-generated by SWIG. I'll add a comment.